src/forklift/core.py

```diff
@@ -254,42 +214,84 @@ def _hash(crate, hash_path):
         fields.append(shape_token)
         att_hash_sub_index = -2
 
-    changes = Changes(fields)
+    changes = Changes(list(fields))
+    changes.table = crate.source
+    changes.fields.append('OID@')
```

Comment:
I add a fields to changes.fields that I don't want to be in the source search cursor. The deleted line passes fields by reference.